bin/node/runtime/src/lib.rs

```diff
@@ -405,6 +418,7 @@ impl pallet_balances::Config for Runtime {
 
 parameter_types! {
 	pub const TransactionByteFee: Balance = 10 * MILLICENTS;
+	pub const OperationalFeeMultiplier: u8 = 5;
```

Comment:
This variable introduced in scope of [transaction priority calculation rework](https://github.com/paritytech/substrate/issues/9317). It regulates priority of operational dispatches (which provide network capabilities).
I guess we can rely he on substrate team leave it as it is.